python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -537,9 +538,14 @@ def is_valid(self):
         """
         if np.dtype(self.ofm) == np.int32 and self.activation is not None:
```

Comment:
Does this mean that a graph with 32 bit OFMs will be offloaded (provided that there is no activation function)?